pkg/jsonnet/decode.go

```diff
@@ -0,0 +1,34 @@
+package jsonnet
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+
+	"github.com/google/go-jsonnet"
+)
+
+// Decoder type can decoce a jsonnet stream into the given output.
```

Comment:
Nitpick: Typo in the comment...fully realizing that the typo is in the `influxdb` repo as well :laugh: